pkg/component/shared/kubeapiserver_test.go

```diff
@@ -241,11 +251,11 @@ var _ = Describe("KubeAPIServer", func() {
 				),
 				Entry("default plugins with overrides",
 					[]gardencorev1beta1.AdmissionPlugin{
-						{Name: "NamespaceLifecycle", Config: &runtime.RawExtension{Raw: []byte("namespace-lifecycle-config")}},
+						{Name: "NamespaceLifecycle", Config: &runtime.RawExtension{Raw: []byte("namespace-lifecycle-config")}, KubeconfigSecretName: pointer.String("secret-1")},
```

Comment:
Why are we specifying `kubeconfigSecretName` for a plugin which is not supported?

Reply:
This is test, doesn't really affect anything here.

Reply:
Still, you could add the test for any of the three supported plugins.